lighthouse/datadog_checks/lighthouse/data/conf.yaml.example

```diff
@@ -20,12 +20,12 @@ instances:
     #
     name: <NAME_PLACEHOLDER>
 
-    ## @param extra_chrome_tags - list - optional
+    ## @param extra_chrome_flags - list - optional
```

Comment:
Hey @ian28223 , can we separate the config change/typo fix from the version bump/release in a separate PR?